server/data/activitiesApiClient.ts

```diff
@@ -101,6 +102,10 @@ export default class ActivitiesApiClient extends AbstractHmppsRestClient {
     })
   }
 
+  postActivityCreation(createBody: ActivityCreateRequest, user: ServiceUser): Promise<void> {
```

Comment:
I guess you dont need it atm - but just for info the api returns the activity model.